docs/governance/steering/charter.md

```diff
@@ -0,0 +1,177 @@
+
+This notification will include:
+
+* a link to the aforementioned GitHub issue
+* the statement providing context on the reason for the vote
+
+There will be a period of two weeks for members of the community to reach out to Steering Committee members to provide feedback.
+
+Community members may provide feedback by the following methods:
+
+* commenting on the GitHub issue
+* sending an email to the
+  [Steering Committee private mailing list][steering-private-list]
+* sending a message to individual committee members
+
+After this feedback period, Steering Committee members must vote on the issue within 48 hours.
+
+If the vote of no confidence is passed, the member in question will be immediately removed from the committee.
+
+## Voting
+
+In the course of the committee's operations, members will be expected to vote on all decisions made within the body's purview.
+
+These votes may be called on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+For public business, the vote must be captured on an issue or pull request.
+
+### Routine business
+
+Unless otherwise specified by a process, the requirement for passing a vote is a **_majority of the [fixed membership of the committee](#composition)_**.
+
+### Abstention
+
+For any self-elected reason, members of the committee may decide to abstain from a vote.
+
+Abstaining members will only be considered as contributing to quorum, in the event that a vote is called in a meeting.
+
+## Meetings
+
+Steering Committee members are generally expected to attend every meeting. We use the following guidelines to determine whether we have reached quorum and are able to proceed with a meeting.
+
+### Quorum
+
+Quorum **to meet** is a **_majority of the [fixed membership of the committee](#composition)_**.
+
+Quorum **to vote in a meeting** is a **_two-thirds supermajority of the [fixed membership of the committee](#composition)_**.
+
+## Inclusive Leadership Training
+
+Members of the committee must take the [Inclusive Open Source Community Orientation](course) course
+in support of our community values.  Members are required to report completion of the course as part of on-boarding within 30 days from the date of their appointment.
+
+## Changes
+
+Committee members may propose a change to this document through the following process:
+
+- Post a pull request to this repository describing the change.
+- Call a public vote for the nearest acceptable business day four (4) weeks after initial 
+  introduction of the change. A vote may be scheduled earlier if all committee members consent.
+- The change is accepted if three-quarters of the committee members vote in favor.
+- The pull request is merged or closed.
+
+## Attribution
+
+This document was adapted from the Kubernetes Steering Committee Charter [[afb3858](upstream)].
```

Comment:
The link isn't redirecting back to https://github.com/kubernetes/steering/blob/afb3858/charter.md